grc20-core/src/mapping/query_utils/query_builder.rs

```diff
@@ -157,6 +160,13 @@ impl MatchQuery {
         self
     }
 
+    pub fn r#rename(mut self, rename: impl Into<Rename>) -> Self {
```

Comment:
You don't need to prefix the method name with `r#` in this case! `r#` is only useful where the name of the method (or variable) is a reserved keyword (e.g.: `r#struct`, `r#type`, `r#where`, etc.)